src/test/e2e/tests/specs.test.ts

```diff
@@ -313,69 +264,48 @@ describe('Verified user', () => {
       it(`should display ${tribunalCourts} results (Tribunal) filter`, async() => {
         expect(await courtNameSearchPage.getResults()).toBe(tribunalCourts);
       });
-
-      it('should click continue to create subscription', async () => {
-        pendingSubscriptionsPage = await courtNameSearchPage.clickContinue();
-        expect(await pendingSubscriptionsPage.getPageTitle()).toEqual('Confirm your subscriptions');
-      });
     });
+  });
 
-    describe('Following the subscription \'search\' by case reference path', () => {
-      const validSearchTerm = 'T485913';
-      const expectedNumOfResults = 1;
-
-      before(async () => {
-        await subscriptionAddPage.open('subscription-add');
-      });
-
-      it('should select \'By case reference number\' option and navigate to search case number page', async () => {
-        await subscriptionAddPage.selectOption('SubscriptionAddByCaseRefNumber');
-        caseReferenceNumberSearchPage = await subscriptionAddPage.clickContinueForCaseReferenceNumberSearch();
-        expect(await caseReferenceNumberSearchPage.getPageTitle()).toEqual('Enter a case reference number');
-      });
-
-      it('should enter text and click continue', async () => {
-        await caseReferenceNumberSearchPage.enterText(validSearchTerm);
-        caseReferenceNumberSearchResultPage = await caseReferenceNumberSearchPage.clickContinue();
-        expect(await caseReferenceNumberSearchResultPage.getPageTitle()).toEqual('Search result');
-      });
-
-      it(`should display ${expectedNumOfResults} results`, async () => {
-        expect(await caseReferenceNumberSearchResultPage.getResults()).toBe(1);
-      });
+  describe('remove subscription', async () => {
+    before(async () => {
+      await subscriptionManagementPage.open('subscription-management');
+    });
 
-      it('should click continue to create subscription', async () => {
-        pendingSubscriptionsPage = await caseReferenceNumberSearchResultPage.clickContinue();
-        expect(await pendingSubscriptionsPage.getPageTitle()).toEqual('Confirm your subscriptions');
-      });
+    it('should click on the first unsubscribe record', async () => {
```

Comment:
Looks like there might be something strange with the subscriptions tests here. Don't believe these should have changed in this ticket so might be a merge conflict issue?